test/WebSites/MvcTagHelpersWebSite/Controllers/MvcTagHelper_HomeController.cs

```diff
@@ -70,5 +72,52 @@ public IActionResult Index()
         {
             return View();
         }
+
+        public IActionResult ProductEdit()
+        {
+            ViewBag.Items = _products;
+
+            return View();
+        }
+
+        public IActionResult Employee()
+        {
+            var employees = new List<Employee>();
+
+            for (int i = 1; i < 5; ++i)
+            {
+                employees.Add(new Employee()
+                {
+                    Name = "EmployeeName_" + i,
+                    Number = i
+                });
```

Comment:
- add `OfficeNumber` and `Gender` settings for some of the `Employee` entries to cover non-default selections.
- reduce the number of entries shown in `ProductEdit` and `Employee` views.  redundancy makes it harder to see what's important here.